tests/impact/client/sal/test_workspace_service.py

```diff
@@ -161,6 +161,12 @@ def test_get_experiments(self, get_all_experiments):
                 ]
             }
         }
+        assert get_all_experiments.adapter.called
+        get_exps_call = get_all_experiments.adapter.request_history[0]
+        assert (
+            f'http://mock-impact.com/api/workspaces/workspace_1/experiments?classPath='
```

Comment:
This does not look right, `?classPath=` should not be in the url when not asking for any specific classpath.